apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -75,8 +86,50 @@ const ERRORS = {
         message: '"qualityControlValue" must be specified if there is "qualityControlAdditionalOptions" or "qualityControlComment"',
         messageForUser: 'api.ticket.QUALITY_CONTROL_VALUE_MUST_BE_SPECIFIED',
     },
+    TICKET_FOR_PHONE_DAY_LIMIT_REACHED: {
+        code: 'BAD_USER_INPUT',
+        type: 'TICKET_FOR_PHONE_DAY_LIMIT_REACHED',
+        message: 'TICKET_FOR_PHONE_DAY_LIMIT_REACHED',
```

Comment:
maybe add a readable message? And `messageForUser`?

Reply:
Nice catch! please add some abstract message for user like: "You reached a daily ticket count limit"

Reply:
As stated in description, the messages will be added once they are ready.